docs/eventing/samples/container-source/README.md

```diff
@@ -0,0 +1,133 @@
+ContainerSource will start a container image which will generate events under contain situations and send messages to a sink uri. ContainerSource could be an easy way to implement your own event sources. This document shows how to configure ContainerSource as event source for functions and gives instructions on how to create an event source using ContainerSource.
```

Comment:
I would add a title at the top.

This sentence feels a bit awkward with a lot of 'ContainerSource'

Reply:
The opening follows the same structure of README of event source samples. I tried to reduce some duplicates of ContainerSource.